<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Photo</class>
 <widget class="QWidget" name="Photo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="QSplitter_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="QVBoxLayout_LeftContainer">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QScrollArea" name="QScrollArea_scroll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="QWidget_scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>575</width>
            <height>695</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QGroupBox" name="QGroupBox_CaptureSettings">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Capture Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="QLabel_Screen">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Select screen to capture</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="QComboBox_ScreenSelection">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="QLabel_CaptureMode">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Select capture mode</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="QComboBox_CaptureMode">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Full screen</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Custom area</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="QLabel_CaptureQuality">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Select capture quality</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="QComboBox_CaptureQuality">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>High</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Low</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="QGroupBox_InformationActions">
             <property name="title">
              <string>Other</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="QHBoxLayout_Rotate">
                <property name="spacing">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="QLabel_RotatePhoto">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Rotate photo</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="QToolButton_90Minus">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-90°</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonIconOnly</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="QToolButton_90Plus">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>90°</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonIconOnly</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="QSpacer_spacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="QHBoxLayout_CreationDate">
                <property name="spacing">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="QLabel_date">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Photo creation date</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="QDateEdit_date">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>105</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="keyboardTracking">
                   <bool>false</bool>
                  </property>
                  <property name="displayFormat">
                   <string>dd-MMMM-yyyy</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="QHBoxLayout_Size">
                <property name="spacing">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="QLabel_Size">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Photo size information</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="QLabel_SizeInfo">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>Resolution 0 x 0 - Size 0 KB</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="QHBoxLayout_Format">
                <property name="spacing">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="QLabel_PhotoFormat">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Photo format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="QLabel_ImageFormat">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="QHBoxLayout_Location">
                <property name="spacing">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="QLabel_Location">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Photo file location</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="QLabel_FilePath">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>No Path detected</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="QSpacer_spacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="QHBoxLayout_Buttons" stretch="1,1,0,0,1,1">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QPushButton" name="QPushButton_OpenImage">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Open Photo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="QPushButton_Capture">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Capture Screen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="QLine_line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="QSpacer_spacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="QPushButton_SaveAs">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Save As</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="QPushButton_ViewInFullScreen">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Full Screen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="QLabel_Thumbnail">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>No photo for previewing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
